benches/tpcc/init_empty.lua

```diff
@@ -0,0 +1,2 @@
+require 'server'
+require 'schema'
```

Comment:
It looks as something internal for the benchmark. Why it is in this repository?

Reply:
All was removed